jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/CloudWatchLogsServiceNode.kt

```diff
@@ -31,7 +31,7 @@ class CloudWatchLogsNode(
     logGroupName,
     AwsIcons.Resources.CloudWatch.LOG_GROUP
 ) {
-    override fun resourceType() = "group"
+    override fun resourceType() = "log-group"
```

Comment:
This will break the actions in the plugin.xml, remember to fix them

Reply:
^ see above lol

Reply:
>.>